lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -861,6 +930,13 @@ SymbolicValue ConstExprFunctionState::getConstantValue(SILValue value) {
 
   // Compute the value of a normal instruction based on its operands.
   auto result = computeConstantValue(value);
+
+  // If this is the top-level lazy interpreter, output a debug trace.
+  if (!fn) {
+    DEBUG(llvm::errs() << "ConstExpr top level: "; value->dump());
```

Comment:
Question: What's the general convention here, `errs` or `dbgs`? I see `DEBUG(llvm::dbgs() << ...)` also and I've been using that. If it's `errs` I'll change mine.

Reply:
Looks like dbgs is most widely used.  I'll adopt it in a follow-on patch. Thanks!